src/backend/tests/fixtures/case_html/with_revoked_probation.html

```diff
@@ -372,26 +372,26 @@
 Court Action: Signed; Court Action Date: 11/08/2010;
 Judge: Judge;
 </i></div><div style="padding-left: 10px"><table cellpadding="0" cellspacing="0" border="0"><tr><th class="ssTableHeaderLabelLeft">Signed:&nbsp;&nbsp;</th><td>11/08/2010</td></tr></table></div><div style="padding-left: 10px;font-size:8pt;">
-            Created: 11/09/2010 12:00 AM</div></td></tr><tr><th class="ssTableHeaderLabel" valign="top" id="RCDER72">11/09/2010</th><td style="border-right: 1px solid black">&nbsp;</td><td>&nbsp;</td><td headers="COtherEventsAndHearings RCDER72"><b>Notice - Advise Appeal Rights</b><div style="padding-left: 10px;font-size:8pt;">
-            Created: 11/09/2010 12:00 AM</div></td></tr><tr><th class="ssTableHeaderLabel" valign="top" id="RCDER73">11/09/2010</th><td style="border-right: 1px solid black">&nbsp;</td><td>&nbsp;</td><td headers="COtherEventsAndHearings RCDER73"><b>Sentence</b><div style="padding-left: 10px;font-size:8pt;">
-            Created: 11/18/2010 12:00 AM</div></td></tr><tr><th class="ssTableHeaderLabel" valign="top" id="RCDER74">11/09/2010</th><td style="border-right: 1px solid black">&nbsp;</td><td>&nbsp;</td><td headers="COtherEventsAndHearings RCDER74"><b>Judgment - Sentence</b>
+            Created: 11/09/2010 12:00 AM</div></td></tr><tr><th class="ssTableHeaderLabel" valign="top" id="RCDER72">11/09/2017</th><td style="border-right: 1px solid black">&nbsp;</td><td>&nbsp;</td><td headers="COtherEventsAndHearings RCDER72"><b>Notice - Advise Appeal Rights</b><div style="padding-left: 10px;font-size:8pt;">
```

Comment:
I find it odd that you're modifying the test source instead of the python tests as the source OECI should never change in theory. But I think this has the same effect so 🤷

Reply:
I know. That was my first idea but, because of other time restrictions on the record the probation revoked date no longer was the limiting date on eligibility. So some edits to the source data would be necessary regardless.